src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/ExecuteRequests/ExecuteStringRequest.cs

```diff
@@ -0,0 +1,27 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.ServiceLayer.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.QueryExecution.Contracts.ExecuteRequests
+{
+    /// <summary>
+    /// Parameters for executing a query directly
+    /// </summary>
+    public class ExecuteStringParams : ExecuteRequestParamsBase
```

Comment:
Ah - no that's fine. it's a small change so as long as we've covered it there it's fine. I just couldn't spot with the GitHub PR review tool